akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingInternalsSpec.scala

```diff
@@ -4,13 +4,17 @@
 
 package akka.cluster.sharding
 
-import akka.actor.{ ExtendedActorSystem, PoisonPill, Props }
+import akka.actor.{ Actor, ExtendedActorSystem, NoSerializationVerificationNeeded, PoisonPill, Props }
+import akka.cluster.sharding.ShardCoordinator.Internal.ShardStopped
 import akka.cluster.sharding.ShardCoordinator.ShardAllocationStrategy
-import akka.testkit.AkkaSpec
+import akka.cluster.sharding.ShardRegion.HandOffStopper
+import akka.testkit.{ AkkaSpec, TestProbe }
 import org.mockito.ArgumentMatchers
 import org.mockito.Mockito._
 import org.scalatest.mockito.MockitoSugar
 
+import scala.concurrent.duration._
+
 class ClusterShardingInternalsSpec extends AkkaSpec("""akka.actor.provider = "cluster"""") with MockitoSugar {
```

Comment:
could you also add port 0 config to this test:
```
  """
  akka.actor.provider = cluster
  akka.remote.netty.tcp.port = 0
  akka.remote.artery.canonical.port = 0
  """
```

Reply:
Done